packages/terra-date-picker/src/DateRange.jsx

```diff
@@ -1,27 +1,34 @@
 import React, { PropTypes } from 'react';
+import moment from 'moment';
 import 'terra-base/lib/baseStyles';
 import DatePicker from './DatePicker';
 
 const propTypes = {
   /**
-   * A moment object to use as the default end date for a date range.
+   * An ISO 8601 string representation of the default end date for a date range.
    */
-  endDate: PropTypes.oneOfType([PropTypes.object]),
+  endDate: PropTypes.string,
   /**
-   * A moment object to use as the default start date for a date range.
+   * An ISO 8601 string representation of the selected start date.
    */
-  startDate: PropTypes.oneOfType([PropTypes.object]),
+  startDate: PropTypes.string,
   /**
    * A callback function to execute when a valid date is selected or entered.
    */
   onChange: PropTypes.func,
 };
 
 class DateRange extends React.Component {
+  static safeMoment(dateISO8601) {
+    const momentDate = moment.utc(dateISO8601);
+
+    return momentDate.isValid() ? momentDate : null;
+  }
 
   constructor(props) {
     super(props);
     this.state = {
+      format: 'MM/DD/YYYY', // TODO: Get the format from i18n
```

Comment:
The format is temporarily hard-coded for now and will be updated to use the format based on the locale using i18n. This format is needed to create a moment object from the ISO 8601 string on line 43.